src/containers/Dashboard.scss

```diff
@@ -1,9 +1,12 @@
 @import 'variables';
 
 .Dashboard {
+  padding-top: 0;
+  //min-height: calc(100vh);
+}
+
+body {
```

Comment:
Keep in the widget-related .scss files only the styles, related to the according widget. Therefore, move `body {...}` to `scss/_scaffolding.scss`. 
The name of the file comes from Bootstrap, where the body-related classes are grouped inside an include with the same name (node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_scaffolding.scss)